gammapy/spectrum/energy.py

```diff
@@ -0,0 +1,204 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import print_function, division
+from astropy.units import Quantity
+from ..utils.array import array_stats_str
+
+import astropy.units as u
+
+
+__all__ = ['Energy',
+           'EnergyBounds']
+
+
+class Energy(Quantity):
+
+    """Energy bin centers.
```

Comment:
How about writing this (following the [Angle](http://astropy.readthedocs.org/en/latest/api/astropy.coordinates.Angle.html) example)?
```
One or more energy value(s).
```